Ginger/Ginger/Reports/HTMLReportAttachmentConfigurationPage.xaml.cs

```diff
@@ -100,26 +100,15 @@ private void OkButton_Click(object sender, RoutedEventArgs e)
                     Reporter.ToUser(eUserMsgKey.FolderNameTextBoxIsEmpty);
                     return;
                 }
-                else if(extraInformationCalculated.Length > 100)
+                else if (extraInformationCalculated.Length > 100)
                 {
                     Reporter.ToUser(eUserMsgKey.FolderNamesAreTooLong);
                     return;
-                }                
-                else
+                }
+                else if (!HasWritePermission(extraInformationCalculated))
```

Comment:
we need to remove also this write permission validation no? how it will validate path exist in Linux? try to add linux path and see if it allows you to proceed